openvmm/openvmm_entry/src/lib.rs

```diff
@@ -2843,7 +2844,7 @@ mod interactive_console {
             arg_id: &str,
         ) -> Vec<String> {
             match (subcommand_path, arg_id) {
-                (["hvlite_entry", "inspect"], "element") => {
+                (["openvmm_entry", "inspect", "openvmm"], "element") => {
```

Comment:
I intended to keep `openvmm_entry` here, but it's also clear I don't know what I'm doing. Is the right thing to keep `openvmm` (the name?) or `openvmm_entry` (the name of the crate?)

Reply:
Turns out, you have to make this `["openvmm", "inspect"]` to keep autocomplete working for members of the inspect tree. Further revision coming.